katalog/prometheus-operator/README.md

```diff
@@ -39,19 +39,20 @@ illustrated in this image from Prometheus Operator repository:
 
```

Comment:
The image linked in the previous line does not exist anymore :-(

Reply:
Fixed. Thank you.